.github/workflows/release-action-docker.yml

```diff
@@ -0,0 +1,56 @@
+name: Release Github action for CLI
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  release-action-yml:
+    name: Release github action for cli and update version in action.yml
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+      DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+      DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          ref: master
+
+      - name: Get version without v character
+        id: version
+        run: |
+          VERSION=${{github.event.release.tag_name}}
+          VERSION_WITHOUT_V=${VERSION:1}
+          echo "value=${VERSION_WITHOUT_V}" >> $GITHUB_OUTPUT
+      - name: Release to Docker
+        run: |
+          echo ${{env.DOCKER_PASSWORD}} | docker login -u ${{env.DOCKER_USERNAME}} --password-stdin
+          npm run action:docker:build
+          docker tag asyncapi/github-action-for-cli:latest asyncapi/github-action-for-cli:${{ steps.version.outputs.value }}
+          docker push asyncapi/github-action-for-cli:${{ steps.version.outputs.value }}
+          docker push asyncapi/github-action-for-cli:latest
+
+      - name: Update action.yml with version
+        run: |
+          VERSION=${{ steps.version.outputs.value }}
```

Comment:
The update to this action.yml won't be a part of the release though if this is done like this.

Reply:
@Shurtu-gal  Yeah, do you want to run this workflow as part of pre-release or this update should be done before we build docker image of github-action?

Reply:
@akshatnema pre-release